### DISCUSSION TITLE
como añadir documentos externos

### DISCUSSION THREAD

**Discussion by davidfv21** [requarks/wiki#7352] [Help / Questions]
como se puede hacer para añadir pdf o documentos word

**Comment by ciltocruz** [comment]
¿Crees que preguntando en español en una plataforma de desarrollo donde el creador de este proyecto seguramente no tenga ni idea, vas a recibir respuesta?

Te respondo igualmente: no se puede mostrar el contenido de ese tipo de documentos. Los puedes añadir como archivo y poner un link a su contenido (que lo descargará). Otra opción es que los subas a Google Drive o plataformas similares y los linkes ahí.

Un saludo.
